.github/merge_rules.yaml

```diff
@@ -519,6 +522,14 @@
   - Lint
   - pull
 
+- name: XPU
+  patterns:
+  - '**xpu**'
+  approved_by:
+  - EikanWang
+  - jgong5
+  - gujinghui
+
```

Comment:
@atalman , @malfet , This sections intends to avoid adding detailed items to merge rule for Intel GPU one by one, just like ROCM. However, it seems like duplicate with XPU ATen section. May I know if there are any side effects?